<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExerciseObject</class>
 <widget class="QWidget" name="ExerciseObject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>41</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
	padding: 0px 0px 0px 5px;
	font: 11pt ;
	background-color: rgb(250, 179, 24);
	color:  rgb(255, 255, 255);
	font: 75 12pt &quot;Comic Sans MS&quot;;
	border-radius: 7px;
}

QLabel#Exc_header{
	background-color: rgb(132, 93, 194);
	font: 14pt ;
	border-top-left-radius: 0;
	border-top-right-radius:;
	border-bottom-left-radius:;
	border-bottom-right-radius:0;
	padding: 0px 0px 0px 0px
;}
Line #l2, #l3, #l5{
	border-radius:0px;
	border: 1px solid rgb(255, 255, 255);
	background-color: rgb(255, 255, 255);
}
#ExerciseFrame{
	background-color:qlineargradient(spread:pad, x1:0.006, y1:0.477273, x2:1, y2:1, stop:0 rgba(95, 95, 95, 34), stop:1 rgba(255, 255, 255, 41))
}
#ExerciseFrame{
	border-top-left-radius: 7; border-top-right-radius:7; border-bottom-left-radius:7; border-bottom-right-radius:7;
}
QPushButton{
	font: 12pt &quot;Comic Sans MS&quot;;
	border: 0px solid 	rgb(132, 93, 194)	 ;
	border-radius: 7px;
	background-color: 	rgb(250, 179, 24)	;
	color: rgb(245, 246, 250);
}
QPushButton:hover{
	background-color: rgb(0, 201, 183)				;
}
QPushButton:pressed{
	
	background-color: rgb(0, 193, 174);
}</string>
  </property>
  <widget class="QLabel" name="cd_exerc">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exerc">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Название</string>
   </property>
  </widget>
  <widget class="QLabel" name="ppts_exerc">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="count_exerc">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>0</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
